aptos-move/framework/aptos-framework/sources/datastructures/big_ordered_map.move

```diff
@@ -151,6 +149,7 @@ module aptos_std::big_ordered_map {
     /// Only allowed to be called with constant size types. For variable sized types,
     /// it is required to use new_with_config, to explicitly select automatic or specific degree selection.
     public fun new<K: store, V: store>(): BigOrderedMap<K, V> {
+        // Use new_with_type_size_hints or new_with_config if your types have variable sizes.
         assert!(
             bcs::constant_serialized_size<K>().is_some() && bcs::constant_serialized_size<V>().is_some(),
             error::invalid_argument(EINVALID_CONFIG_PARAMETER)
```

Comment:
not sure if it is better to allow new() for all types - and then if first values inserted are not representative - by a factor of 100x+ - not being able to insert bigger values later.
or as implemented here - be conservative - and ask people to intentionally provide or not provide size hints.